frame/assets/src/types.rs

```diff
@@ -56,16 +67,8 @@ pub struct AssetDetails<Balance, AccountId, DepositBalance> {
 	pub(super) approvals: u32,
 	/// Whether the asset is frozen for non-admin transfers.
 	pub(super) is_frozen: bool,
```

Comment:
Can be also recorded as a status

Reply:
Yeah, that's true. But i think i'll rather explore that in a different PR.